pages/api/webhooks.js

```diff
@@ -12,21 +12,15 @@ export const config = {
   }
 };
 
-const buffer = (req) => {
-  return new Promise((resolve, reject) => {
-    const body = [];
-    req
-      .on('data', (chunk) => {
-        body.push(chunk);
-      })
-      .on('end', () => {
-        resolve(Buffer.concat(body));
-      })
-      .on('error', (err) => {
-        reject(err);
-      });
-  });
-};
+async function buffer(readable, encoding) {
```

Comment:
Any reason you included the `encoding` here? The way we use the method I think `encoding` would be undefined. For `Buffer` it defaults to `utf8`, so couldn't we further simplify by omitting `encoding` completely?

Reply:
Thank you for your feedback 🙏

Yes, we can, absolutely! I just thought could be useful one day, but we could just omit it for sure.